lib/modules/dosomething/dosomething_campaign/includes/CampaignController.php

```diff
@@ -2,6 +2,15 @@
 
 class CampaignController {
 
-  // @TODO: handle main app controller-y things such as CRUD in here.
+  /**
+   * Update the specified Campaign in storage.
+   *
+   * @param  int  $id
+   * @return void
+   */
+  function update($id) {
+    cache_clear_all('campaigns_api_request', 'cache_dosomething_api', TRUE);
```

Comment:
tricky, but at the minimum this needs to be scoped only to campaign nodes https://github.com/DoSomething/phoenix/pull/6126/files#diff-ae398bb8864fc3c26317f7ccaef04c0aR1263